app/data/__init__.py

```diff
@@ -2,20 +2,25 @@
 from ..services.location.csbs import CSBSLocationService
 from ..services.location.jhu import JhuLocationService
 from ..services.location.nyt import NYTLocationService
+from ..services.location import LocationService
 
-# Mapping of services to data-sources.
-DATA_SOURCES = {
-    "jhu": JhuLocationService(),
-    "csbs": CSBSLocationService(),
-    "nyt": NYTLocationService(),
-}
+class DataSource:
+    __DATA_SOURCES = {}
 
+    def __init__(self):
+        self.__DATA_SOURCES['jhu'] = JhuLocationService()
+        self.__DATA_SOURCES['csbs'] = CSBSLocationService()
+        self.__DATA_SOURCES['nyt'] = NYTLocationService()
```

Comment:
I appreciate that you took the time to explain what you were doing and why.
But I still don't really see the value of this refactor.

For much the same reason as I said in your other PR.
https://github.com/ExpDev07/coronavirus-tracker-api/pull/349#discussion_r671771042

`DataSource` doesn't need to be a class. What is the point of creating a `DataSource` instance considering all the instances have the exact same sources?

There are ways to make the `__init__` method return the same instance singleton, but this won't do that.
Although I'm not actually recommending that (just an FYI).